tox.ini

```diff
@@ -1,11 +1,12 @@
 [tox]
-envlist = py27, flake8
```

Comment:
Do we still need py27 here?

Reply:
I don't know, if anyone wants to use py27 after py3 integrated.

Reply:
It's not a question of wanting, it's a question of needing....but no, I think we can remove py27

Reply:
if this was a request to drop it from here, I just added another commit